src/ClassicUO.Assets/AnimationsLoader.cs

```diff
@@ -130,76 +130,73 @@ private unsafe void LoadInternal()
                 LoadUop();
             }
 
-            if (UOFileManager.Version >= ClientVersion.CV_500A)
-            {
-                string path = UOFileManager.GetUOFilePath("mobtypes.txt");
+            string path = UOFileManager.GetUOFilePath("mobtypes.txt");
 
-                if (File.Exists(path))
+            if (File.Exists(path))
+            {
+                var typeNames = new string[5]
                 {
-                    var typeNames = new string[5]
-                    {
-                        "monster",
-                        "sea_monster",
-                        "animal",
-                        "human",
-                        "equipment"
-                    };
+                    "monster",
+                    "sea_monster",
+                    "animal",
+                    "human",
+                    "equipment"
+                };
+
+                using (var reader = new StreamReader(File.OpenRead(path)))
+                {
+                    string line;
 
-                    using (var reader = new StreamReader(File.OpenRead(path)))
+                    while ((line = reader.ReadLine()) != null)
```

Comment:
Note for another PR when it makes sense: There are new UTF8/ReadOnlySpan<byte> convenience methods for parsing data that we can use to avoid allocations.